### DISCUSSION TITLE
When i go to error page then go back to home page the `useFetch` didn't request in server but rather client.

### DISCUSSION THREAD

**Discussion by WenisOK** [nuxt/nuxt#20301] [Questions]
I have a nuxt blog project. I have a `useFetch` method in `pages/index.vue`, When I request such as `/test` and I not define this page router. It's suceess show the nuxt template error page. I click the `go back home` button, The `useFetch` method in  `index.vue`  doesn't request in server, But client requested, and client console throw the CORS error message .

**Comment by warflash** [comment]
Might help if you provide a reproduction [here](https://nuxt.new/) - that makes things easier to follow 🙏

**Comment by ndragun92** [comment]
@WenisOK only first-page render makes a server request. Any further client navigation is performing client requests. If u really want to use server request instead of nuxt-link u would have to use a href that reloads the page but in that case, u lose nuxt magic. But for error page in general u can use a href when u redirect user to a home page then u will make a fresh request to server
